microsoft-edge/webview2/toc.yml

```diff
@@ -36,6 +36,8 @@
   href: roadmap.md
 - name: How to
   items:
+    - name: Link Static Loader
```

Comment:
Is this a "Link static loader" or a "Static link loader" or something else?

Reply:
It is (How to) link the static loader, I just removed "the"

Reply:
Actually, the loader is what we are trying to link, in a static manner (with the package), does this make sense? I think "Statically link loader" works?

Reply:
Updated.